src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double firstParallel;
+    private final double secondParallel;
+    private final int height;
```

Comment:
For consistency, consider using 'double' instead of 'int' for the height, since the other dimensions are represented as doubles.